tests/ProxyManagerTest/ProxyGenerator/RemoteObject/MethodGenerator/RemoteObjectMethodTest.php

```diff
@@ -19,10 +18,30 @@
  */
 final class RemoteObjectMethodTest extends TestCase
 {
+    /**
+     * @var string
+     */
+    private const STATIC_CODE = <<<PHP
+\$reflectionMethod = new \\ReflectionMethod(__CLASS__, __FUNCTION__);
+\$args = \\func_get_args();
+foreach (\\array_slice(\$reflectionMethod->getParameters(), \\count(\$args)) as \$param) {
+            /**
```

Comment:
Secondary, but alignment is a bit wonky here

Reply:
Not used anymore, changed tests to use inline block codes as you were doing before.